pkg/persistence/config/loader/config_loader.go

```diff
@@ -86,25 +114,21 @@ func LoadConfig(fs afero.Fs, context *LoaderContext, filePath string) ([]config.
 	return configs, nil
 }
 
-func parseFile(fs afero.Fs, filePath string) ([]persistence.TopLevelConfigDefinition, error) {
-	data, err := afero.ReadFile(fs, filePath)
-	if err != nil {
-		return nil, err
-	}
+func parseFile(data []byte) ([]persistence.TopLevelConfigDefinition, error) {
 
 	definition := persistence.TopLevelDefinition{}
-	err = yaml.UnmarshalStrict(data, &definition)
+	err := yaml.UnmarshalStrict(data, &definition)
 
 	if err != nil {
 		if strings.Contains(err.Error(), fmt.Sprintf("field config not found in type %s", persistence.GetTopLevelDefinitionYamlTypeName())) {
```

Comment:
this is no longer needed, you explicitly check for the 'config' field on line 69-73 now